ci_tools_atomic_dex/src/dependencies.nim

```diff
@@ -2,6 +2,7 @@ import osproc
 import vcpkg
 
 let g_packages = [
+    (name: "jsoncpp", head: false),
```

Comment:
This is spurious, the library for nlohmann json in vcpkg is https://github.com/microsoft/vcpkg/tree/master/ports/nlohmann-json